ProcessLib/TwoPhaseFlowWithPP/TwoPhaseFlowWithPPLocalAssembler-impl.h

```diff
@@ -7,6 +7,31 @@
  *
  */
 
+/**
+* common nomenclature
+* --------------primary variable----------------------
+* pn_int_pt    pressure for nonwetting phase at each integration point
+* pc_int_pt    capillary pressure at each integration point
+* --------------secondary variable--------------------
+* temperature              capillary pressure
+* Sw wetting               phase saturation
+* dSw_dpc                  derivative of wetting phase saturation with respect
+* to capillary pressure
+* rho_nonwet               density of nonwetting phase
+* drhononwet_dpn           derivative of nonwetting phase density with respect
+*to nonwetting phase pressure
+* rho_wet                  density of wetting phase
+* k_rel_nonwet             relative permeability of nonwetting phase
+* mu_nonwet                viscosity of nonwetting phase
+* lambda_nonwet            mobility of nonwetting phase
+* k_rel_wet                relative permeability of wetting phase
+* mu_wet                   viscosity of wetting phase
+* lambda_wet               mobility of wetting phase
+* _pressure_wet            output vector for wetting phase pressure with respect
+* to each integration point
+* _saturation              output vector for wetting phase saturation with
```

Comment:
Move the documentation of private members to their declaration's place. Also mention, what you have in another comment, about the secondary variables output....